pkg/plugin/pr-sanitizer/cmd/main.go

```diff
@@ -42,6 +42,11 @@ func (gh *GitHubLabelsEventsHandler) HandleEvent(log log.Logger, eventType githu
 	return nil
 }
 
+// GhClient created while event handler initialization which is used to communicate with the GitHub API
+func (gh *GitHubLabelsEventsHandler) GhClient() ghclient.Client {
+	return gh.Client
+}
```

Comment:
this is Java way of doing it. We should use a different approach to solve the problem